ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml

```diff
@@ -162,6 +175,14 @@ tests:
       FEATURE_SET: CustomNoUpgrade
     workflow: openshift-e2e-aws-sharednetwork
   timeout: 6h0m0s
+- as: periodics-e2e-aws-ovn-shared-vpc
+  steps:
```

Comment:
Does this not have `interval: 8h` on purpose?

Reply:
...just making sure you're paying attention! :D